stumpy/gpu_mpdist.py

```diff
@@ -85,6 +87,9 @@ def gpu_mpdist(
     ...         device_id=all_gpu_devices)
     0.00019935236191097894
     """
-    MPdist = core._mpdist(T_A, T_B, m, gpu_stump, percentage, k, device_id=device_id)
+    partial_gpu_stump = functools.partial(gpu_stump)
```

Comment:
Don't we need to add `T_X_subseq_isconstant` to this function?

Reply:
We do. I am planning to do it in this PR after taking care of the current issues and moving some test functions to test_core.